### DISCUSSION TITLE
Opsgenie: Add Option to send Tags without Key

### DISCUSSION THREAD

**Discussion by jozahner** [grafana/grafana#42807] [Alerting]
We use Opsgenie heavily with other tools like Prometheus. Any other tool has an option to send "Tags" without the key, so only the value. In our case the key is not needed and adds just unneeded information. 
Please include a general option for the Opsgenie Notification Channel where we can select whether we want to send only the key of a tag or the tag plus the key (as today). Issue is shown below
![Grafana_Opsgenie_Config](https://user-images.githubusercontent.com/33521474/144980214-8edbc7de-5649-4863-a3d2-0e911bef2122.png)
:
![Grafana_Opsgenie_FR](https://user-images.githubusercontent.com/33521474/144980237-3e0523f4-6f2d-4c08-bf6d-d6fd4a6a2026.png)

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
